storybook/stories/components/BaseIframe/BaseIframe.js

```diff
@@ -8,7 +8,7 @@ export const BaseIframeTemplate = ({ title, url, height }) => {
     ${height ? `height="${height}"` : ``}
     allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
     allowfullscreen="allowfullscreen"
-    class="BaseIframe"
+    class="w-full border-none"
```

Comment:
I'm not sure we even need these as Tailwind classes. We're setting the `width=100%` attribute above and all the browsers I can check easily show no difference if I get rid of them. Am I missing some reason why they're important?

Reply:
By default, iframes are styled with a border, but our reset css (part of Tailwind) overrides that. So I think we can get rid of that one.

For width, I think it would be more useful if we did something like the following, in the cases where users can't always edit their iframe code directly:

```html
<div class="BaseIframe">
  <iframe src="..." />
</div>
```

```scss
.BaseIframe iframe {
  @apply w-full;
}
```

This would, of course, no longer deprecate the `.BaseIframe` class or eliminate scss. Thoughts on the above?

Reply:
Hmm… 

> in the cases where users can't always edit their iframe code directly

Do you have an example of when this would be the case?

Even if it is, I'm not sure if we should feel obligated to keep that CSS in our system to account for it. It ought to be fairly easy for a user in that situation to handle that themselves.

Reply:
> Even if it is, I'm not sure if we should feel obligated to keep that CSS in our system to account for it. It ought to be fairly easy for a user in that situation to handle that themselves.

Good point. We aren't providing "just in case" styles for other components either, so I think it makes sense to remove this. I'll update the branch.

Reply:
> Do you have an example of when this would be the case?

I'm thinking of when a user is copying iframe code from a service but isn't familiar with the code. Or if an embed is generated from oEmbed. We account for this in our own implementation of Explorer 1, so I think it's okay to leave out from the design system, and devs can resolve it how they see fit in their own implementations.